github/spec/features/validate_repos_spec.rb

```diff
@@ -18,7 +18,7 @@
 
     stub_repos_json(repos)
 
-    expect { ValidateRepos.new(@octokit_mock).verify_repo_tags }.to output("\e[0;32;40mUntagged govuk repos: No mismatches found.\e[0m\n\n\e[0;32;40mFalsely tagged govuk repos: No mismatches found.\e[0m\n").to_stdout
+    expect { ValidateRepos.new(@octokit_mock).verify_repo_tags }.to_not output.to_stdout
```

Comment:
Nitpick: I'd suggest doing the remove of colorize and the change of behaviour (i.e. no output when everything is OK) in its own commit, explaining that in the next commit you'll be running this from an automated build, so need a more parseable output for the build process.

Currently from reading the PR and commit message you wouldn't know that the behaviour of the underlying script has been changed.

Reply:
This still stands @MuriloDalRi 🙏 thanks

Reply:
It's hard to pick only the parts that remove colorize because they're changing half a line and also the code for the method changed quite a bit. I'll just add more info to the same commit and hopefully that's enough (I doubt anyone has actually tried to run this on their machine and will notice any difference anyway)